packages/react/src/utils/_responsive.scss

```diff
@@ -11,3 +11,9 @@
     @content;
   }
 }
+@mixin media($name, $tokens: $themes) {
+  $breakpoint: breakpoint($name, $tokens);
+  @media (max-width: #{$breakpoint}) {
+    @content;
+  }
+}
```

Comment:
I do not see any difference with the `container` mixin ?

Reply:
The `container` mixin is using `@container` css media query where here we want to use `@media`

Reply:
Oh my bad !